.github/workflows/docker-image.yml

```diff
@@ -19,11 +19,15 @@ on:
           - 'gravitino-playground:hive'
           - 'gravitino-playground:ranger'
           - 'gravitino-iceberg-rest-server'
+      docker_repo_name:
+        description: 'Docker repository name (default is apache)'
+        required: false
+        default: 'apache'
+        type: string
```

Comment:
I think we didn't need this, Please remove it, thanks.

Reply:
@xunliu  I think this is needed, without this, developers can not test the `Publish Docker image` workflow at their own Github repo and Docker repo, since it always push newly built image to `apache` Docker repo.

![Screenshot 2025-05-08 at 8 34 08 AM](https://github.com/user-attachments/assets/b5a81175-4c39-465f-9b6c-69d305b9e895)